rootfs/Dockerfile

```diff
@@ -1,4 +1,14 @@
-FROM alpine:3.1
+FROM alpine:3.3
+
+# install git and configure gituser
+ENV GITHOME /home/git
+ENV GITUSER git
+# this is so the minio client (https://github.com/minio/mc) works properly
+ENV DOCKERIMAGE=1
+ENV DEIS_RELEASE 2.0.0-dev
```

Comment:
If there's a critical security issue we can purge the docker cache in Travis and force a rebuild. That way we can fix it in master before we cut a new release. :)

Let's move it to the bottom